cnf-certification-test/accesscontrol/suite.go

```diff
@@ -581,7 +581,7 @@ func TestPodTolerationBypass(env *provider.TestEnvironment) {
 			// Check if the tolerations fall outside the 'default' and are modified versions
 			if tolerations.IsTolerationModified(t) {
 				podsWithRestrictedTolerationsNotDefault = append(podsWithRestrictedTolerationsNotDefault, put.String())
-				tnf.ClaimFilePrintf("Pod: %s has been found with non-default %s toleration %s which is not allowed.", string(t.Effect), put.String(), t.Effect)
+				tnf.ClaimFilePrintf("Pod: %s has been found with non-default %s toleration %s which is not allowed.", put.Data.Name, put.String(), t.Effect)
```

Comment:
The stringer for put (put.String()) is already showing the name of the pod, so that first "put.Data.Name" is not needed.
```
func (p *Pod) String() string {
	return fmt.Sprintf("pod: %s ns: %s",
		p.Data.Name,
		p.Data.Namespace,
	)
}
```
Are you sure this new version of the trace is ok?

Reply:
I'll refactor since we can use the .String()